### PR TITLE
Add with-react-native example

### PR DESCRIPTION
**Pull request by yankovalera**
## Summary

Add with-react-native example

### CODE REVIEW

File: examples/with-react/src/pages/PokemonList.js [hunk 1]
```diff
@@ -16,19 +17,9 @@ const PokemonList = () => {
...
-
-      {data && (
-        <ul>
-          {data.pokemons.map((pokemon) => (
-            <li key={pokemon.id}>{pokemon.name}</li>
-          ))}
-        </ul>
-      )}
-    </div>
+    <View>
```
Comment (JoviDeCroock) [inline examples/with-react/src/pages/PokemonList.js:20]: I think this isn't intended?

File: examples/with-react-native/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "react-native": "0.64.0",
+    "urql": "^2.0.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "@babel/runtime": "^7.12.5",
+    "babel-jest": "^26.6.3",
+    "metro-react-native-babel-preset": "^0.64.0"
+  },
+  "jest": {
```
Comment (JoviDeCroock) [inline examples/with-react-native/package.json:24]: We can remove jest

File: examples/with-react/src/pages/PokemonList.js (hunk 1 above)
Comment (yankovalera) [reply examples/with-react/src/pages/PokemonList.js]: oops

**Review by JoviDeCroock — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -17,4 +17,9 @@ exchanges/*/LICENSE
 tmp/
 dist/
 examples/*/public
-examples/*/yarn.lock
\ No newline at end of file
+examples/*/yarn.lock
+examples/*/ios/
```
Comment (kitten) [inline .gitignore:21]: Since these are one offs I'd maybe just add them to the React Native example & create a new gitignore there 🤔

Comment (yankovalera) [reply .gitignore:21]: I'd prefer them here in case we add more RN examples later, which could be the case for e.g. handling auth, etc.

**Pull request merged**